frontend-service/src/main/java/com/amazonaws/blox/frontend/mappers/DeleteEnvironmentMapper.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazonaws.blox.frontend.mappers;
+
+import com.amazonaws.blox.dataservicemodel.v1.model.wrappers.DeleteEnvironmentRequest;
+import com.amazonaws.blox.dataservicemodel.v1.model.wrappers.DeleteEnvironmentResponse;
+import com.amazonaws.blox.frontend.operations.DeleteEnvironment;
+import com.amazonaws.serverless.proxy.internal.model.ApiGatewayRequestContext;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+@Mapper(uses = EnvironmentMapper.class)
```

Comment:
Do we need a unit test for the mapper (even though this is quite simple)?

Reply:
Little bit tricky here to unit test the DeleteEnvironmentMapper. 
tl;td    The mapper test is inside the `DeleteEnvironmentTest` controller test.

I personally prefer to have unit test per one class only. As you may see example like `StartDeploymentMapper`.  However, the `DeleteEnvironmentMapper` is a little bit different. This mapper depends on the `EnvrionmentMapper`, and we are using spring model to generate the code,  we need to load those mappers into spring context to wire them up.

Instead of having other Spring runner to test the mapper only,  I refactored current controller unit test a little bit, so the mapper will be tested with the controller.

Reply:
Right, I can see that in order to test the mapper we need to wire up the EnvironmentMapper. I think doing it via the controller is fine. As long as there's a way to test the mapper it should be fine.